src/features/chat/Chat.module.css

```diff
@@ -0,0 +1,28 @@
+.colLeft {
```

Comment:
I think we can use tailwindcss classes for these styles.

Reply:
I don't see suitable standard tw class for min-with. Added custom to tw config
By the way, tw config had duplicate entries (maxWidth obj, spacing.100px, etc) Fixed it too